__tests__/endpoint.test.js

```diff
@@ -67,9 +67,30 @@ describe("/api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: 11
         });
-      });
+    });
   });
+  test('GET:200 We still recieve an array of articles when comment count is 0 - test the join', () => {
+
+    return request(app)
+    .get("/api/articles/7")
+    .expect(200)
+    .then(({ body }) => {
+      const { article } = body;
+      expect(article).toEqual({
+        author: 'icellusedkars',
```

Comment:
Do you use prettier? Some consistency with single and double quotes would be nice